tests/output_equivalence.rs

```diff
@@ -119,7 +119,7 @@ fn make_tests() -> Vec<Trial> {
         .map(|param| {
             Trial::test(
                 format!("addr2line {}", param.as_ref().map_or("", String::as_str)),
-                move || run_test(param.as_ref().map(String::as_str)),
+                move || run_test(param.as_deref()),
```

Comment:
Can you also change the previous line to use `as_deref`?

Reply:
Sure, done!